doc/api/tls.md

```diff
@@ -186,11 +185,17 @@ of an application. The `--tls-cipher-list` switch should by used only if
 absolutely necessary.
 
 ## Class: tls.Server
+<!-- YAML
+added: v0.11.3
+-->
```

Comment:
v0.3.2 if my git fu deceives me not.  It's from one of the v0.3.x releases at any rate.